template/Dockerfile

```diff
@@ -38,27 +38,27 @@ COPY . /app/
 # for your build, add them to this file and the gitkube remote, then re-apply the remote using `kubectl apply -f cms.remote.yaml`.
 # If you need to build the Docker image elsewhere, you can specify them via `--build-arg <name>=<value>` for the `docker build` command.
 # See https://docs.docker.com/engine/reference/builder/#using-arg-variables for further details
-ARG REACT_APP_API_BASE_URL
-ARG REACT_APP_DEPLOYMENT_ENV
-ARG REACT_APP_BASE_NAME
+ARG REACT_APP_API_BASE_URL=/api
```

Comment:
These `ARG`s should be static (or removed completely) and entirely replace by `ENV` inject during runtime. This way, we no longer need to build different docker images (via the drone `matrix` definitions) for every environment (e.g. dev, staging, prod).

Reply:
The ARG statements are just sane default values and can all be overwritten via ENV.

Reply:
Then we shouldn't rely on build-args anyways + `ENV` overwrite should only be possible during runtime, not compile time.

Reply:
I agree.